src/sage/algebras/hecke_algebras/cubic_hecke_algebra.py

```diff
@@ -699,19 +696,14 @@ class CubicHeckeAlgebra(CombinatorialFreeModule):
     Element construction::
 
         sage: ele = CHA3.an_element(); ele
-        (-w)*s1*s2^-1 + v*s1 + u*s2 + ((-v*w+u)/w)
+        -w*s1*s2^-1 + v*s1 + u*s2 - ((v*w-u)/w)
         sage: ele2 = ele**2; ele2
-        w^2*(s1^-1*s2)^2 + (-u*w^2)*s1^-1*s2*s1^-1 + (-v*w)*s2*s1^-1*s2
-        + (-v*w^2)*s1^-1*s2^-1 + u*w*s1*s2*s1^-1*s2 + (-u*w)*s1^-1*s2*s1
-        + (-u*v*w+2*v*w-2*u)*s1*s2^-1 + u*v*w*s2*s1^-1 + u*v*s2*s1 + v^2*w*s1^-1
-        + (-u^2*w)*s1*s2*s1^-1 + ((u*v^2*w-2*v^2*w-u*w^2+2*u*v)/w)*s1
-        + u*v*s1*s2 + (u^2*w+v^2*w)*s2^-1 + ((u^3*w-2*u*v*w+2*u^2)/w)*s2
-        + ((-u^2*v*w^2-v^3*w^2+v^2*w^2-2*u*v*w+u^2)/w^2)
+        w^2*(s1^-1*s2)^2 - u*w^2*s1^-1*s2*s1^-1 - v*w*s2*s1^-1*s2 - v*w^2*s1^-1*s2^-1 + u*w*s1*s2*s1^-1*s2 - u*w*s1^-1*s2*s1 - (u*v*w-2*v*w+2*u)*s1*s2^-1 + u*v*w*s2*s1^-1 + u*v*s2*s1 + v^2*w*s1^-1 - u^2*w*s1*s2*s1^-1 + ((u*v^2*w-2*v^2*w-u*w^2+2*u*v)/w)*s1 + u*v*s1*s2 + (u^2*w+v^2*w)*s2^-1 + ((u^3*w-2*u*v*w+2*u^2)/w)*s2 - ((u^2*v*w^2+v^3*w^2-v^2*w^2+2*u*v*w-u^2)/w^2)
         sage: B3 = CHA3.braid_group()
```

Comment:
I meant formatting like here.

Reply:
Will fix it eventually.

Problem is… I'm not sure if it's worth it because nobody reads all the detail anyway and sage-fixdoctests breaks it. I guess using an autoformatter to wrap the lines is fine (the test output checker ignores line breaks I think)

(I might have broken line break in too many places in https://github.com/sagemath/sage/pull/39027 also, oops)

Reply:
I don't understand...

Look at https://doc-pr-39018--sagemath.netlify.app/html/en/reference/algebras/sage/algebras/hecke_algebras/ariki_koike_algebra#sage.algebras.hecke_algebras.ariki_koike_algebra.ArikiKoikeAlgebra.LT.product_on_basis

You introduced a long output line in the doctest.